lib/postgrex.ex

```diff
@@ -364,7 +358,7 @@ defmodule Postgrex do
 
   ## Example
 
-      {:ok, res} = Postgrex.transaction(pid, fn(conn) ->
+      {:ok, res} = Postgrex.transaction(conn, fn(conn) ->
```

Comment:
Please leave this as `pid` because if it is a pid then the `conn` in `fn(conn)` won't be `pid` and the shadowing of `conn` is confusing. This occurs elsewhere too.